src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/RisksTest.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import java.time.LocalDateTime
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds.Risks as ArnRisks
+
+class RisksTest : DescribeSpec(
+  {
+    describe("#toRisks") {
+      it("maps ARN Risks to Integration API Risks") {
```

Comment:
This is pretty picky so not fussed normal test naming convention for 1-1 attributes is: "maps one-to-one attributes to x attributes"

Reply:
Ty done